graphql-cli/src/main/java/io/ballerina/graphql/exception/ConfigTypesGernerationException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.graphql.exception;
+
+import io.ballerina.graphql.cmd.DiagnosticMessages;
+import io.ballerina.graphql.cmd.GraphqlDiagnostic;
+import io.ballerina.graphql.cmd.Utils;
+import io.ballerina.tools.diagnostics.DiagnosticSeverity;
+
+/**
+ * Exception type definition for Ballerina utils code generation related errors.
+ */
+public class ConfigTypesGernerationException extends GenerationException {
+    private String message;
+
+    public ConfigTypesGernerationException(String message, Throwable e) {
+        super(message, e);
+        this.message = message;
+    }
+
+    public ConfigTypesGernerationException(String message) {
+        super(message);
+        this.message = message;
+    }
+
+    public String getDiagnosticMessage() {
+        GraphqlDiagnostic graphqlDiagnostic = Utils.constructGraphqlDiagnostic(
+                DiagnosticMessages.GRAPHQL_CLI_110.getDescription(),
```

Comment:
Let's define a new diagnostic message and use it here

Suggestion
```
GRAPHQL_CLI_112("GRAPHQL_CLI_112", "Ballerina config types code generation related error occurred.",
            DiagnosticSeverity.ERROR);
```
Add here
https://github.com/ballerina-platform/graphql-tools/blob/270270199edafd5ec619c34f86319058288597a0/graphql-cli/src/main/java/io/ballerina/graphql/cmd/DiagnosticMessages.java#L50

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/commit/080517c00268a6547d19d178c0aa9ea573d38156